packages/@lwc/template-compiler/src/codegen/helpers.ts

```diff
@@ -273,10 +273,10 @@ export function styleMapToStyleDeclsAST(styleMap: { [name: string]: string }): t
     const styles: Array<[string, string] | [string, string, boolean]> = Object.entries(
         styleMap
     ).map(([key, value]) => {
-        const important = value.endsWith('!important');
+        const importantRegex = /\s*!\s*important\s*$/;
```

Comment:
```suggestion
        const importantRegex = /\s*!\s*important\s*$/i;
```

I looked into it, and this is actually case-insensitive ([codepen](https://codepen.io/nolanlawson-the-selector/pen/GRaavKZ)).

Could we add some tests for `!IMPORTANT` as well please?